confignode/src/main/java/org/apache/iotdb/confignode/manager/partition/RegionGroupStatus.java

```diff
@@ -29,12 +29,19 @@ public enum RegionGroupStatus {
    */
   Available("Available"),
 
+  /**
+   * All Regions in RegionGroup are in the Running, Unknown or ReadOnly status, and at least 1 node
+   * is in ReadOnly status, the number of Regions in the Unknown or ReadOnly status is less than
+   * half
+   */
+  Discouraged("Discouraged"),
+
   /**
    * The following cases will lead to Disabled RegionGroup:
    *
-   * <p>1. There is a Region in ReadOnly or Removing status
+   * <p>1. There is a Region in Removing status
```

Comment:
Maybe not exactly, if user is migrating one region, is the RegionGroup Disabled?